src/components/Details.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+function Region({
+  name, Confirmed, allDeaths, newCases, allRecovered,
+}) {
+  return (
+    <li className="detailsRow">
+      <div>
+        <h3>{name}</h3>
+      </div>
+      <div>
+        <h4>
+          Confirmed:
+          {Confirmed}
+        </h4>
+        <h4>
+          Deaths:
+          {allDeaths}
+        </h4>
+        <h4>
+          Open Cases:
+          {newCases}
+        </h4>
+        <h4>
+          Recovered:
+          {allRecovered}
+        </h4>
+      </div>
+    </li>
+  );
+}
+
+Region.propTypes = {
+  name: PropTypes.string.isRequired,
+  Confirmed: PropTypes.number.isRequired,
+  allDeaths: PropTypes.number.isRequired,
+  newCases: PropTypes.number.isRequired,
+  allRecovered: PropTypes.number.isRequired,
+};
+
+function Regions() {
+  const countries = useSelector((state) => state.covidReducers);
+  const { id } = useParams();
+  let country = [];
+  countries.forEach((element) => {
+    if (element.name === id) {
+      country = element;
+    }
+  });
+
+  const {
+    name, regions, cases_confirmed, total_deaths, open_cases, recovered,
+  } = country;
+
+  return (
+    <div>
+      <div className="detailsBaner">
+        <img
+          className="map2"
+          src="https://cdn.pixabay.com/photo/2020/04/29/07/54/coronavirus-5107715_960_720.png"
+          alt="map"
+          /* eslint-disable no-param-reassign */
+          onError={(event) => {
+            event.target.src = 'https://img.icons8.com/pastel-glyph/120/000000/map.png';
+            event.onerror = null;
+          }}
+        />
+        <div className="countryStatus">
+          <h2>{name}</h2>
+          <h4>
+            Today Confirmed:
+            {' '}
+            {cases_confirmed}
```

Comment:
- [ ] Kindly fix the error shown in the dev tools, take a look at the image below, I know this is because the data is undefined.

![undefined](https://user-images.githubusercontent.com/53124548/165926088-dc873c1f-9327-4110-a8cb-13ce52f91a7d.png)